src/Adapter/AutoRouteRefreshCommandAdapterInterface.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2017 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\RoutingAutoBundle\Adapter;
+
+/**
+ * Class RefreshOrmCommand.
```

Comment:
this comment is wrong, but i am in favor of not having any comment at all unless there is something relevant to say.

please check the method doc below, i think we could explain a bit more precesily what the methods do.

Reply:
I agree with you.